integration/gateway/gateway_suite_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright IBM Corp All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package gateway
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/hyperledger/fabric-protos-go/common"
+	gateway "github.com/hyperledger/fabric-protos-go/gateway"
+	"github.com/hyperledger/fabric-protos-go/peer"
+	"github.com/hyperledger/fabric/bccsp/sw"
+	"github.com/hyperledger/fabric/common/flogging"
+	"github.com/hyperledger/fabric/integration"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/internal/pkg/comm"
```

Comment:
Do we need to use an explicitly internal package in the outside-in integration tests instead of the grpc package?

Reply:
Now using PeerClientConn so this will go